website/docs/r/policy_edge_high_availability_profile.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "FIXME"
```

Comment:
`Beta` or `Fabric`

Reply:
I think that all those policy edge should be beta, right?

Reply:
If we expect those to be fully tested by release, then we can go with `Fabric` right away